watchface

```diff
@@ -23,6 +23,7 @@ Available options:
 -w or --wall WP sets the wallpaper for deploy or test to the named file
 
 Available commands:
+update          use git to update the unofficial-watchfaces project
```

Comment:
I think it would be clearer to new users to write "use git to update your local copy of the unoffical-watchfaces repository.".
Since the current sentence could imply to update the upstream "project"?

Reply:
done!